src/odin/utils/logger.py

```diff
@@ -22,12 +22,10 @@ def free_disk_bytes() -> int:
 MdValues = Optional[Union[str, int, float]]
 
 DATE_FORMAT = "%Y-%m-%dT%H:%M:%S%z"
-LOG_FORMAT = "{asctime} {levelname:>8s} {message}"
-# if running on AWS drop timestamp because Splunk handles that
-if bool(os.getenv("AWS_DEFAULT_REGION")):
-    LOG_FORMAT = "{levelname:>8s} {message}"
```

Comment:
This check can stay in. It is needed to print timestamp information to the console when running the application not in AWS.

Reply:
k, reverted this change